### PR TITLE
Make Logos Adapt to Color Theme

### PR DESCRIPTION
**Pull request by KarateBrot**
- Make logos adapt to color theme
- Fix sponsor logos
- Small layout changes to the BF logo on the landing page

# Before
grafik
grafik

# After
grafik
grafik

### CODE REVIEW

**Comment by SupaflyFPV** [comment]
looks great, only thing is the Axis logo rather dominates if it resizes according to height?

**Comment by KarateBrot** [comment]
> looks great, only thing is the Axis logo rather dominates if it resizes according to height?

I was not able to fix this. Something seems to overwrite the appearance. Maybe I can figure it out later if no one beats me to it

**Review by haslinghuis — COMMENTED** [review]
As configurator gets the content from cloud API does this solve the inverted image for Axis too?

For dark mode the Axis logo does not work with `#315` checked out?

image

image

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Axis logo does not work because it is capitalized.

image

**Comment by KarateBrot** [comment]
> @haslinghuis:
> Axis logo does not work because it is capitalized.

While working on the PR I renamed it TWICE but it still didn't register. And while trying to fix it, it still didn't want to register the change. Probably a git bug. I now got it working by first pushing a commit with the deleted file and then re-added the file 🙄
Interestingly enough, my browser displayed the logo correctly even though the file name was capitalized.

> @haslinghuis:
> As configurator gets the content from cloud API does this solve the inverted image for Axis too?

No, this is completely independent from the configurator source files. At least it should be.

**Comment by ctzsnooze** [comment]
Looks really good, except that the logos suddenly get huge when the screen is narrow enough in the Relations:Sponsors page.  It looks like this:

Screen Shot 2024-01-02 at 20 39 49

In contrast, the logos on the main landing page don't have that problem.

Also the row of three images further down the page does not scale smaller very well, getting too big for small screens..

Please don't remove the duplicate copy of `sponsors.md` at src/Pages/.  This is needed since it will get displayed by the outgoing link from Configurator.  If you remove it, people clicking on the sponsor icon in Configurator will go to betaflight.com/sponsors and be returned `Page not found`.  PR #310 solves this problem in a different way, but has not yet been merged.  There is another way using a plugin.  Until one of these are merged, we need to keep a copy of sponsors.md in the  src/Pages/ directory.

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [inline docs/relations/sponsors.mdx:10]: changing `flex-col` to `flex-cols-4` and removing `grid md:grid-cols-4` fixes the behaviour with very small screen widths.

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [inline docs/relations/sponsors.mdx:14]: HQprop now seems too small; changing `w-⅘` to `w-full`, balances it better with the others.

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [inline docs/relations/sponsors.mdx:21]: radiomaster image now seems larger than before; `w-4/5` seems to work better than `w-full`.

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [inline docs/relations/sponsors.mdx:38]: Dogcom might be better at 4/5ths size than `w-full`

**Comment by ctzsnooze** [comment]
Also we can perhaps better balance the sizes of the images... with the changes to the images, HQ seems smaller than before, and radiomaster is more dominant than before.

Current appearance
Screen Shot 2024-01-02 at 21 21 11

With the following suggested changes:
- HQ to ⅘ from full
- radiomaster from full to ⅘
- dogcom from full to ⅘
it looks like th
Screen Shot 2024-01-02 at 21 18 11
is:

**Review by haslinghuis — APPROVED** [review]
Let's merge the fix - we need our focus on firmware and client software as we can always improve documentation after release.

**Comment by ctzsnooze** [comment]
It is trivial for @KarateBrot to fix this PR so that it works nicely at small screen sizes.

It is unfair on HQProp to make their logo significantly smaller in visual impact than the others.
It is not OK that at small screen sizes the 'fix' fails to look good.  That could affect all phone users.
These two things need to be fixed.

It is trivial for @KarateBrot to fix those issues in this PR, and I already explained how to do it.  

However if he is short of time, we can merge and I'll make a followup PR to implement those fixes.  Hence I will approve on that basis.

**Review by ctzsnooze — DISMISSED** [review]

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [reply docs/relations/sponsors.mdx:10]: Please ignore the previous comment, must have been an image caching problem; sizing of this image now seems OK with code in the PR.

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [reply docs/relations/sponsors.mdx:14]: Perhaps it was a caching error, but the image sizes now appear OK after rebooting and force-refreshing.
Please ignore the previous suggestion.

File: docs/relations/sponsors.mdx
Comment (ctzsnooze) [reply docs/relations/sponsors.mdx:21]: Perhaps it was a caching error, but the image sizes now appear OK after rebooting and force-refreshing.
Please ignore the previous suggestion.

**Comment by ctzsnooze** [comment]
Sorry PLEASE DO NOT MERGE unless this PR restores a duplicate of `docs/relations/sponsors.mdx` in `pages/` so that we do not break the click on the link in configurator.
Or else merge PR #310 which means you won't have to do that.

**Comment by KarateBrot** [comment]
@ctzsnooze I added the sponsors page again and let it fetch the content to prevent the need to maintain duplicates. Feel free to change the styling of whatever you don't like in this PR. This PR is more about making the logos adapt to the color theme.

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
PR tests out 100% perfectly now.   The bug on squashing to very narrow screen sizes has been fixed.  Looks great in both night and day modes.
Approved, and ready to merge, I think.

**Review by ctzsnooze — APPROVED** [review]

**Comment by nerdCopter** [comment]
@KarateBrot  `git mv -f OldFileNameCase newfilenamecase` might have worked as well.

unsure what operating system you run. Windows is case-insensitive mostly.  there is also the git variable : `git config [--global] core.ignorecase [true|false]`

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
